service/controller/key/key.go

```diff
@@ -20,10 +20,17 @@ const (
 	LabelAppOperatorVersion = "app-operator.giantswarm.io/version"
 	LabelManagedBy          = "giantswarm.io/managed-by"
 	LabelServiceType        = "giantswarm.io/service-type"
+	LabelKVMOperator        = "kvm-operator.giantswarm.io/version" // TODO: Why isn't this in apiextensions?
```

Comment:
This has been added to apiextensions -- will be removed from here once apiextensions is released